### ISSUE TITLE
Does not run on FreeBSD due to Linux dependency: /proc/mounts

### ISSUE THREAD

**Issue by borjam** [Graylog2/graylog2-server#12542]
## Expected Behavior
The previous versions didn't rely on the presence of /proc/mounts, which is not supported on FreeBSD's procfs.
It worked perfectly on FreeBSD

## Current Behavior
Graylog aborts on startup with this message

```

2022-04-27 14:45:30,466 WARN o.u.FileUtil [main] File not found or not readable: /proc/mounts
Exception in thread "main" java.lang.ArithmeticException: / by zero
	at org.graylog2.shared.system.stats.fs.OshiFsProbe.lambda$fsStats$11(OshiFsProbe.java:116)
	at java.util.stream.Collectors.lambda$toMap$58(Collectors.java:1321)
	at java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169)
	at java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:440)
	at java.util.HashMap$EntrySpliterator.forEachRemaining(HashMap.java:1723)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:566)
	at org.graylog2.shared.system.stats.fs.OshiFsProbe.fsStats(OshiFsProbe.java:106)
	at org.graylog2.bootstrap.preflight.DiskJournalPreflightCheck.checkJournalDirSizeAndType(DiskJournalPreflightCheck.java:67)
	at org.graylog2.bootstrap.preflight.DiskJournalPreflightCheck.runCheck(DiskJournalPreflightCheck.java:62)
	at com.google.common.collect.ImmutableList.forEach(ImmutableList.java:422)
	at org.graylog2.bootstrap.preflight.PreflightCheckService.runChecks(PreflightCheckService.java:51)
	at org.graylog2.bootstrap.ServerBootstrap.runPreFlightChecks(ServerBootstrap.java:135)
	at org.graylog2.bootstrap.ServerBootstrap.beforeInjectorCreation(ServerBootstrap.java:126)
	at org.graylog2.bootstrap.CmdLineTool.run(CmdLineTool.java:292)
	at org.graylog2.bootstrap.Main.main(Main.java:45)

```

## Possible Solution
Does it need to use /proc/mounts for any useful purpose?

## Steps to Reproduce (for bugs)
Starting it on FreeBSD 13.1-RC4.

## Context
Can't run this version of Graylog on FreeBSD. Previous versions worked perfectly.
## Your Environment

* Graylog Version: 4.3.0-rc3
* Java Version: OpenJDK openjdk8-8.312.07.1
* Elasticsearch Version: 7.10.2
* MongoDB Version: 3.6.23
* Operating System: FreeBSD 13.1-RC4
* Browser version: Safari 15.4 (irrelevant anyway)
*

**Comment by mpfz0r** [comment]
Thanks for the report. I've recently seen this error in a different context. I will try to make this code more robust.
In the meantime you can use
`disable_native_system_stats_collector=true` in your graylog.conf as a workaround

**Comment by borjam** [comment]
Thank you, testing now!
Let me know how I can help to find the required statistics on FreeBSD. Maybe it's possible to achieve it in a portable way.

**Comment by borjam** [comment]
And confirmed, it started up. 

Thanks!

**Comment by chalfling** [comment]
Also got this error on FreeBSD 12.2 and Graylog 4.3.0

**Comment by dch** [comment]
thanks, warning added to the FreeBSD port for this as well.

**Comment by mpfz0r** [comment]
This will be fixed in the upcoming 4.3.2 release.
